include/cantera/transport/TransportBase.h

```diff
@@ -751,6 +788,15 @@ class Transport
     std::weak_ptr<Solution> m_root;
 };
 
+
+//! Empty transport object used for exception handling
+class NoTransport : public Transport
+{
+    virtual std::string transportType() const {
+        return "None";
+    }
```

Comment:
In C++ it may still be best to stick with `”None”`?

Reply:
Sorry, GH trimmed off the relevant part of this code snippet. I wasn't questioning the string `"None"` versus the Python builtin symbol `None`. This comment is on the `NoTransport` class, and I was wondering if it would be simpler to just have `Transport::transportType` return this value rather than adding a special new class to represent the lack of a transport model.

Reply:
:+1: ... makes sense. Adopted!